dev/integration_tests/windows_startup_test/lib/main.dart

```diff
@@ -0,0 +1,64 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:ui' as ui;
+
+import 'package:flutter/services.dart';
+import 'package:flutter_driver/driver_extension.dart';
+
+void drawHelloWorld() {
+  final ui.ParagraphStyle style = ui.ParagraphStyle();
+  final ui.ParagraphBuilder paragraphBuilder = ui.ParagraphBuilder(style)
+    ..addText('Hello world');
+  final ui.Paragraph paragraph = paragraphBuilder.build();
+
+  paragraph.layout(const ui.ParagraphConstraints(width: 100.0));
+
+  final ui.PictureRecorder recorder = ui.PictureRecorder();
+  final ui.Canvas canvas = ui.Canvas(recorder);
+
+  canvas.drawParagraph(paragraph, ui.Offset.zero);
+
+  final ui.Picture picture = recorder.endRecording();
+  final ui.SceneBuilder sceneBuilder = ui.SceneBuilder()
+    ..addPicture(ui.Offset.zero, picture)
+    ..pop();
+
+  ui.window.render(sceneBuilder.build());
+}
+
+void main() async {
+  // Create a completer to send the result back to the integration test.
+  final Completer<String> completer = Completer<String>();
+  enableFlutterDriverExtension(handler: (String? message) => completer.future);
+
+  try {
+    const MethodChannel methodChannel =
+        MethodChannel('tests.flutter.dev/windows_startup_test');
+
+    final bool? visible = await methodChannel.invokeMethod('isWindowVisible');
+    if (visible == null || visible == false) {
```

Comment:
Maybe just `if (visible != true) {`?

Reply:
I would love to do `if (visible != true)` but there's a [Dart lint that warns if you implicitly handle the null](https://dart.dev/tools/linter-rules#use_if_null_to_convert_nulls_to_bools); they recommend something like `if (!(visible ?? false))` or `if ((visible ?? false) == false)` which I find too terse. This felt like the most reasonable compromise. Does that seem reasonable? Or should I ignore that lint?

Reply:
Ah yeah -- I had a vague recollection of something in our style guide I vehemently disliked about these sorts of checks. This is better than either of those. No worries then :)